src/pages/main/MainPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import Navbar from '../../components/Navbar/Navbar';
+import headerData from '../../data/headerData';
+import Footer from '../../components/Footer/Footer';
+import Beers from '../../components/Beers/Beers';
+import Contact from '../../components/Contact/Contact';
+
+const MainPage = () => (
+  <div>
+    <Helmet>
+      <title>
+        {headerData.name}
+        {' '}
+        | Beer Lovers Application
+      </title>
+      <meta name="description" content="Beer lovers application" />
+    </Helmet>
+    <Navbar />
+    <Beers />
+    <Contact />
+    <Footer />
```

Comment:
[REQUIRED] As suggested by the previous reviewer, kindly make the beer types shaped in a two-column grid and the background colour is staggered like a chessboard to match the given design. When you look at Cindy'sdesign you notice that there are no spaces. between the items.